src/gpt-auto-generator/gpt-auto-generator.c

```diff
@@ -182,6 +183,9 @@ static int add_cryptsetup(const char *id, const char *what, bool rw, bool requir
         }
 
         return 0;
+#else
+        return log_error_errno(SYNTHETIC_ERRNO(EOPNOTSUPP), "Encrypted partition, please set -Dlibcryptsetup=true to enalbe support");
```

Comment:
Typo!

Reply:
Also, `-Dlibcryptsetup=true` is not a good user-facing message. How is the user supposed to know that we're talking about meson? Maybe something like
"Partition is encrypted, but the project was compiled without libcryptsetup support."